class-udm-updater.php

```diff
@@ -66,6 +67,12 @@ public function __construct($mothership, $muid, $relative_plugin_file, $options
 		// Prevent updates from wordpress.org showing in all circumstances. Run with lower than default priority, to allow later processes to add something.
 		add_filter('site_transient_update_plugins', array($this, 'site_transient_update_plugins'), 9);
 
+		$this->udmupdaterl10n = array(
+			'duplicate_site_id1' => esc_js(__("Your attempt to connect to the updates server was unsuccessful due to duplicate site IDs. However, the server has disconnected the site that relates to the detected site ID. Please connect again to claim this site and to receive updates.", 'udmupdater')),
```

Comment:
@andtown I'm not sure this message is accurate? As I read the code of the server plugin, there's no need to try again; if you see this, then the licence is now assigned to this site.

Reply:
@DavidAnderson I'm not sure if that's entirely correct, when it succeeds in connecting to the updates server, the plugin server will then replace the entitlement URL with the new URL of the cloned site. But, since it only replaces the URL and doesn't replace the site ID, which is the ID from the original site, so we first need to disconnect the plugin from the updates server and manually remove the site ID from the `wp_options` in the database, and then recreate the site ID by once again connecting the plugin to the updates server. Looking at the code, that is the only way to recreate the site ID if it was detected being duplicated, also we can recreate a new site ID only if we've first detected a duplicate site ID.

Reply:
@andtown I understand - I think. But I can't find the original Trello card. Does what you say above as what you're intending to do describe the same thing as the card, or something different?

Reply:
@DavidAnderson684 Yes it does, it still fits and is still the same thing as what is stated on the card, it's just that due to complexity some part has not yet been explained on the card, and what I say above is the one I found while implementing the fix.

Reply:
@andtown Rather than telling the user to click the button to connect again, why not just send that HTTP request automatically?

Reply:
@DavidAnderson684 Ahh right, it can be done that way too. I think the user will get confused if there are too many warnings/information, which actually have the same meaning. So, I've implemented this one, and now when someone with his cloned site connects to the updates server, he will receive the below alert message:

![image](https://user-images.githubusercontent.com/400912/106455540-e85e7380-64be-11eb-8930-873acaa83cdc.png)

there will be another message that appears in the connector box right above the login form saying that `this site was previously sharing a single licence also used another site (URL), probably because of being duplicated - this resulted in it being disconnected`. This is still the same message that is described in the Trello card. Due to the changes, this PR has two PR dependencies which are https://github.com/DavidAnderson684/simba-plugin-manager-updater/pull/12 and https://github.com/DavidAnderson684/simba-plugin-manager-updater/pull/14